src/node.js

```diff
@@ -36,6 +37,7 @@ class ProcessGraphNode {
 		let args = Utils.mapObjectValues(this.arguments, arg => Utils.isObject(arg) && typeof arg.toJSON === 'function' ? arg.toJSON() : arg);
 		return Object.assign({}, this.source, {
 			process_id: this.process_id,
+			namespace: this.namespace,
```

Comment:
`|| null` needed?

Reply:
No, undefined is also fine, which will remove the property.